.circleci/config.yml

```diff
@@ -1,71 +1,68 @@
 version: 2
 jobs:
-  # Build and deploy- Just for master
-  build-and-deploy:
+  build:
     docker:
-      - image: circleci/node:lts
-      
-    working_directory: ~/repo
-
+      - image: circleci/buildpack-deps:buster
+    environment:
+        BUILD_IMAGE: italia/publiccode-editor-build
```

Comment:
Cannot use `italia/publiccode-editor-build:$CIRCLE_SHA1` here because CircleCI doesn't expand it.